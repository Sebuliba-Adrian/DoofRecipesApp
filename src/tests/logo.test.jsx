import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../components/logo';

it('renders correctly', () => {
    const logo = renderer.create(<Logo />).toJSON();
    expect(logo).toMatchSnapshot();
});
