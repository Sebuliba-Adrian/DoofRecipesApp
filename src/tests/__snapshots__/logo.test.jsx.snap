// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h3
    className="logo text-center mb-3"
  >
    Doof Recipes App
  </h3>
  <p
    className="grey-text text-center mb-5"
  >
    Record your favourite recipes 
  </p>
</div>
`;
